examples/main.tf

```diff
@@ -82,3 +82,54 @@ resource "instaclustr_vpc_peering" "example_vpc_peering" {
   peer_account_id = "1234567890"
   peer_subnet = "10.0.0.0/20"
 }
+
+resource "instaclustr_cluster" "example-elasticsearch" {
```

Comment:
Could the 2 new examples here be reduced to 1?

Reply:
Yep done.

Reply:
done